manager/integration/tests/test_recurring_job.py

```diff
@@ -476,13 +479,96 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     13. Verify that during the recurring backup, the volume's frontend is
         disabled, and pod cannot start.
     14. Wait for the recurring backup finishes.
-        Delete the recurring backup
-    15. In a 6-mintute retry loop, verify that the pod can eventually start.
-
-    16. Change the setting allow-recurring-job-while-volume-detached to false
-    17. cleanup
+        Delete the recurring backup.
+    15. In a 10-minutes retry loop, verify that the pod can eventually start.
```

Comment:
Wow. I takes 10 mins. Seems like a significant downtime.

Reply:
Changed back 6 minutes in the original test plan.

Reply:
Oops, It is good. I think 10 mins is good! It actually take longer than I thought.

Reply:
@c3y1huang Does it require 10 minutes? or 6 minutes is good enough. If 6-minute is good enough, we don't need to update it to 10 minutes.

Reply:
@yasker I've tried with 6 minutes but occasionally will fail on time-out. Wait for 10 minutes gives more stable results and it's I see it's also the default time wait for other tests.